admin/requirements.in

```diff
@@ -1,5 +1,6 @@
+cffi>=1.16.0
```

Comment:
This is just a transitive dependency right? We shouldn't specify it in the `.in` file then.

Reply:
IIRC if not specified, coz the requirements are being built on an earlier Python version, it will pull in an earlier version not compatible with 3.11. If there's a better way to pin the version I'm down to use it.

Reply:
I think (but don't know in detail) that the correct way would be a constraints file, but that's something to look in separate from this PR:

https://pip.pypa.io/en/latest/user_guide/#constraints-files

Reply:
https://github.com/freedomofpress/securedrop/issues/7127, resolving for now

Reply:
See also freedomofpress/securedrop-client#1538.